app/TerminalViewController.m

```diff
@@ -368,6 +368,16 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     }
 }
 
+- (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection {
```

Comment:
Why not add a traitCollectionDidChange to the various views and view controllers that need appearance updates?

Reply:
I picked something that's always in the view hierarchy.